tests/test_user.py

```diff
@@ -218,6 +217,48 @@ def test_noop(self, requests_mock, cli_runner):
         cli_runner.invoke(change_command, ["1"])
         assert not requests_mock.called
 
+    @pytest.mark.usefixtures("mock_change_password_url")
```

Comment:
Making fixture for this is not needed, when you are using the call only in two functions. Keep the requests_mock directly in the functions only.